core/frontend/src/tile/map/ImageryTileTree.ts

```diff
@@ -53,9 +53,18 @@ export class ImageryMapTile extends RealityTile {
 
   public selectCartoDrapeTiles(drapeTiles: ImageryMapTile[], rectangleToDrape: MapCartoRectangle, drapePixelSize: number, args: TileDrawArgs): TileTreeLoadStatus {
     // Base draping overlap on width rather than height so that tiling schemes with multiple root nodes overlay correctly.
-    if (this.isLeaf || (this.rectangle.xLength() / this.maximumSize) < drapePixelSize || this._anyChildNotFound) {
-      if (this.isDisplayable && !this.isNotFound && !this.isOutOfLodRange)
+    const isSmallerThanDrape = (this.rectangle.xLength() / this.maximumSize) < drapePixelSize;
+    if (  this.isLeaf           // Include leaves so tiles get stretched past max LOD levels.
+      || isSmallerThanDrape
+      || this._anyChildNotFound) {
+      if (this.isOutOfLodRange) {
+      }

```

Comment:
If you intend to keep this if as an empty clause, maybe add a comment...

Reply:
Seems to be a leftover, will remove it.